spec/simplecov_helper.rb

```diff
@@ -1,20 +0,0 @@
-require "active_support/inflector"
-require "active_model/type"
-require "simplecov"
-module SimpleCovHelper
-  def self.process_coverage
-    SimpleCov.start do
-      add_filter "config/initializers/"
-      add_filter "spec/"
-      add_filter "services/migration_helpers/"
-      add_filter "config/environments/"
-
-      minimum_coverage(line: 100)
-      merge_timeout 3600
```

Comment:
This line didn't make it into the new config, because it wasn't actually doing anything beforehand.

`SimpleCov.collate` ignores timestamps by default.